lib/game.rb

```diff
@@ -0,0 +1,50 @@
+require_relative './player.rb'
+require_relative './board.rb'
+require_relative './ui.rb'
+
+class Game
+  include UI
+  def initialize
+    board = Board.new
```

Comment:
As I mentioned above, instantiating an object inside another one isn't a good practice. Pass an already instantiated Board object to Game